src/ansible_navigator/ui_framework/curses_defs.py

```diff
@@ -22,3 +23,18 @@ class CursesLinePart(NamedTuple):
 
 CursesLine = Tuple[CursesLinePart, ...]
 CursesLines = Tuple[CursesLine, ...]
+
+
+RgbTuple = Tuple[int, ...]
```

Comment:
Can it really be arbitrarily many `int`s? Isn't it just a 3-tuple?

Reply:
have a look....